docker-compose.yml

```diff
@@ -3,20 +3,18 @@ version: '3.1'
 services:
   redis:
     image: redis
-    ports: ['6379:6379']
     healthcheck:
       test: ["CMD", "redis-cli","ping"]
 
   postgres:
     image: postgres:16
-    ports: ['5432:5432']
     restart: always
     environment:
       POSTGRES_USER: app
       POSTGRES_DB: app_development
       POSTGRES_PASSWORD: secret
     volumes:
-    - pgdata:/var/lib/postgresql/data
+    - hostedgpt_pgdata:/var/lib/postgresql/data
```

Comment:
This was getting prefixed by the directory name but if you repeat directory names then you can get a conflict on the volume name on the host machine.